crates/core/component/dex/src/component/position_manager.rs

```diff
@@ -570,32 +569,6 @@ pub(crate) trait Inner: StateWrite {
             "updating position assets' aggregate balances"
```

Comment:
the update of the aggregate balance was for debugging the value circuit breaker; this tracing message and the `net_change_*` variables are likely no longer needed